docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -303,7 +308,7 @@ Step 2: Disk Formatting
 
    Choose one of the following options:
 
-   - Unencrypted::
+   - Unencrypted (choose this if you want per-user encyprtion later on)::
```

Comment:
nit: typo in encryption, but I'd probably just do "Unencrypted or Per-User Encrypted Home Directories"